File: 30days.c
#include <stdio.h>
#include <string.h>
#include <math.h>
#include <stdlib.h>

// Day 0: Hello, World.
int main() {
   char inputString[105]; // declare a variable to hold our input
   scanf ("%[^\n]%*c", inputString); // get a line of input from stdin and save it to our variable
  
   // Your first line of output goes here
   printf("Hello, World.\n");

   printf("%s\n",inputString);

   return 0;
}

File: 30days.js
//Day 0: Hello, World.
function processData(inputString) {
    // Your first line of output goes here
    process.stdout.write("Hello, World.");

    // Write the second line of output
    var input = inputString;
    process.stdout.write("\n" + input)
}


process.stdin.resume();
process.stdin.setEncoding("ascii");
_input = "";
process.stdin.on("data", function (input) {
    _input += input;
});

process.stdin.on("end", function () {
    processData(_input);
});